### DISCUSSION TITLE
This repository is no longer active for open source contributions and is for profit by taking money from companies to build UI.

### DISCUSSION THREAD

**Discussion by Gitstar-OC** [magicuidesign/magicui#372] [General]
The recent issue are not being reviewed or approved by anyone and this adds to a question if this is still active or not. I would like to be a member of this magicui team and help people by maintaining it.

**Comment by dillionverma** [comment]
Hey @Gitstar-OC! The repository is indeed still active! We're a small team and had to prioritize the pro templates for a few months so that we can continue maintaining and delivering value to the community. 

We are now getting back to improving the oss repo. What are you interested in helping out with?

**Reply by Gitstar-OC** [reply]
Hey @dillionverma I think it's more than 3 months when I started this discussion and  I also know it's hard to maintain an oss.  

I know you guys are a group of great coders and need to maintain this repository, what I wanted to ask is while you code and help with the pro templates I help with maintaining and checking the whole issues, discussions and pr's and merge if needed and they look good. 

By this way Magic UI will be the leading stylish components library.

**Comment by Yeom-JinHo** [comment]
I’m helping review issues and PRs for Magic UI at the moment.

If you’d like to collaborate on anything or need input on a specific thread, feel free to tag me anytime 🙂
